README.md

```diff
@@ -20,6 +20,7 @@ module.exports = {
                     "workerInterval": "30",
                     "rotateInterval": "0 0 * * *",
                     "rotateModule": true,
+                    "roleAttached": false,
```

Comment:
Can you add documentation. How would some one define the IAM role in his instance. Along with aws reference.